### PR TITLE
Export: you have to specify what you want to export

### PR DESCRIPTION
**Pull request by SteffenBrinckmann**

### CODE REVIEW

**Comment by SteffenBrinckmann** [comment]
@jmurugan-fzj This is complete from my side: I do not understand why the test fails. Can you help?

**Comment by jmurugan-fzj** [comment]
> @jmurugan-fzj This is complete from my side: I do not understand why the test fails. Can you help?

@SteffenBrinckmann This is failing due to the changes you made in exportELN siganture in the previous PR, I shall create a new PR for the tests today, give me sometime (y)

**Comment by SteffenBrinckmann** [comment]
@jmurugan-fzj  Then we would merge both PRs at the same time? Or would it be better to merge this PR, then you create a new PR, which we merge thereafter?

File: pasta_eln/gui.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def __init__(self) -> None:
     super().__init__()
     venv = ' without venv' if sys.prefix == sys.base_prefix and 'CONDA_PREFIX' not in os.environ else ' in venv'
     self.setWindowTitle(f"PASTA-ELN {__version__}{venv}")
-    self.setWindowState(Qt.WindowMaximized)  # type: ignore
+    # self.setWindowState(Qt.WindowMaximized)  # type: ignore
```
Comment (jmurugan-fzj) [inline pasta_eln/gui.py:44]: Can we remove this comments if not needed?

File: pasta_eln/inputOutput.py [hunk 2]
```diff
@@ -526,6 +526,17 @@ def testELNFile(fileName:str) -> None:
   Args:
     fileName (str): file name of .eln to test
   """
+  import sys
+  try:
+    sys.path.append('/home/steffen/FZJ/DataScience/Repositories/TheELNConsortium/TheELNFileFormat/tests/')
```
Comment (jmurugan-fzj) [inline pasta_eln/inputOutput.py:531]: What's this strange path in production code?

File: pasta_eln/inputOutput.py [hunk 3]
```diff
@@ -526,6 +526,17 @@ def testELNFile(fileName:str) -> None:
...
+    from test_00_pypi_rocrate         import testFile as roCrateTest
```
Comment (jmurugan-fzj) [inline pasta_eln/inputOutput.py:532]: why do we have test methods/modules imported here?

**Review by jmurugan-fzj — CHANGES_REQUESTED** [review]

**Comment by jmurugan-fzj** [comment]
> @jmurugan-fzj Then we would merge both PRs at the same time? Or would it be better to merge this PR, then you create a new PR, which we merge thereafter?

@SteffenBrinckmann Could you please review my PR quickly, this will fix the pytest errors and you can work on the pylint errors as part of this PR or separately...!

**Comment by SteffenBrinckmann** [comment]
@jmurugan-fzj now it should be good

**Comment by jmurugan-fzj** [comment]
> @jmurugan-fzj now it should be good

@SteffenBrinckmann Could you address the comments also quickly?

**Comment by SteffenBrinckmann** [comment]
@jmurugan-fzj Sorry, I had overlooked those. Thanks, they made much sense.

**Review by jmurugan-fzj — APPROVED** [review]

**Pull request merged**